easybuild/easyblocks/g/gcc.py

```diff
@@ -289,6 +289,26 @@ def configure_step(self):
         - set platform_lib based on config.guess output
         """
 
+        sysroot = build_option('sysroot')
+        if sysroot:
+            # based on changes made to GCC in Gentoo Prefix
+            # https://gitweb.gentoo.org/repo/gentoo.git/tree/profiles/features/prefix/standalone/profile.bashrc
+
+            # add --with-sysroot configure option, to instruct GCC to consider
+            # $EPREFIX as the root filesystem of the operating system
```

Comment:
$EPREFIX?? too much C&P ??

Reply:
I was using `os.getenv('EPREFIX')` when testing this rather than `build_option('sysroot')`, I'll clean up the comment...